config/spryk/spryks/Spryker/Glue/BackendApi/Plugin/AddGlueBackendApiResourcePlugin.yml

```diff
@@ -0,0 +1,90 @@
+spryk: template
+description: "Adds a new GlueBackendApi Resource Plugin."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    resource:
+        description: "Defines the resource name this resource plugin will work with."
+
+    className:
+        value: "{{ resource }}Resource"
+
+    extends:
+        value: \Spryker\Glue\GlueApplication\Plugin\GlueApplication\AbstractResourcePlugin
+
+    implements:
+        value: \Spryker\Glue\GlueRestApiConventionExtension\Dependency\Plugin\RestResourceInterface
+
+    template:
+        value: Common/Model.php.twig
+
+    targetFilename:
+        value: "{{ resource }}Resource.php"
+
+    targetPath:
+        value: "src/{{ organization }}/Glue/{{ module }}/Plugin/"
+
+    application:
+        value: Glue
+
+    layer:
+        inherit: true
+        default: Plugin
+
+postSpryks:
+    - AddMethod:
```

Comment:
Please add a new postSpryk `AddImplements`. This one will be used two times here with conditions.

```
# Add interface to the Resource when used with API Type REST
- AddImplements:
    condition: "apiType === 'REST'"
    arguments:
        interface:
            value: \Spryker\Glue\GlueRestApiConventionExtension\Dependency\Plugin\RestResourceInterface
        target:
            value: "\\{{ organization }}\\Glue\\{{ module }}\\Plugin\\{{ className }}"
# Add interface to the Resource when used with API Type JSON
- AddImplements:
    condition: "apiType === 'JSON'"
    arguments:
        interface:
            value: \Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\JsonApiResourceInterface
        target:
            value: "\\{{ organization }}\\Glue\\{{ module }}\\Plugin\\{{ className }}"